docs/source/EM_analyzers/stator_wdg_res.rst

```diff
@@ -1,42 +1,61 @@
-Conductor area is found as :math:`A_\text{cond} = K_\text{Cu}A_\text{slot}/z_Q`, where :math:`A_\text{slot}` is a stator slot area and :math:`K_\text{Cu}` is a slot fill factor. The length of a coil consists of two main parts:
+where :math:`z_Q` is a number of turns per coil, :math:`z_C` is a number of coils per phase, :math:`l_\text{coil}` is an average length of a single loop of a coil, and :math:`\sigma_\text{cond}` and :math:`A_\text{cond}` are conductor conductivity and cross-section area. 
+Conductor area is found as :math:`A_\text{cond} = K_\text{Cu}A_\text{slot}/(n_\text{layers}z_Q)`, where :math:`A_\text{slot}` is a stator slot area, :math:`K_\text{Cu}` is a slot fill factor, and :math:`n_\text{layers}` is a number of layers (:math:`n_\text{layers}=1` for a single-layer winding and :math:`n_\text{layers} = 2` for a double-layer winding).
+
+To calculate :math:`l_\text{coil}`, the following coil shape is assumed
+
+.. figure:: ./Images/coil_diagram.svg
+   :alt: coil_diagram 
+   :align: center
+   :width: 500 
+
+This is the view when one side of the stator is cut along the axial direction and unrolled. Using this diagram, :math:`l_\text{coil}` is calculated as
 
 .. math::
 
     l_\text{coil} &= 2(l_\text{st} + l_\text{end wdg})\\
 
-Here, :math:`l_\text{st}` is a length of a coil along the axial length of a motor and :math:`l_\text{end wdg}` is an end winding length. A factor of two is included to take into account a returning coil side.
+where :math:`l_\text{st}` is a length of a coil along the axial length of a motor; :math:`l_\text{end wdg} = l_1 + 2l_2` is an end winding length.
 
-Equation that is used to estimate :math:`l_\text{end wdg}` is shown below:
+The segments with length :math:`l_2` represent the bent parts of a coil and are calculated as 1/4th of the circle circumference. The diameter of this circle is estimated as the average of stator tooth width :math:`w_\text{st}` and the length between adjacent slots :math:`\tau_u`.
 
 .. math::
 
-    l_\text{end wdg} &= \frac{1}{2} \pi \frac{\tau_u + w_\text{st}}{2} + \tau_u K_\text{ov} (y - 1)\\
+    l_2 &= \frac{1}{4} \pi \frac{\tau_u + w_\text{st}}{2}\\
 
-This equation consists of two main parts.
-The first part is a length of bent parts of a coil end, which is evaluated as a length of a half-circle with a diameter :math:`(\tau_u + w_\text{st})/2`. 
-Here, :math:`w_\text{st}` is a stator tooth width and :math:`\tau_u` is an arc length between adjacent slots evaluated at a median depth of a slot:
+Here, :math:`\tau_u` is calculated as the length of an arc (when a stator is in its normal cylindrical shape) at a median depth of a slot:
 
 .. math::
 
-    \tau_u &= \frac{2 \pi}{Q} (r_\text{si} + d_\text{sp} + d_\text{st})\\
+    \tau_u &= \frac{2 \pi}{Q} (r_\text{si} + d_\text{sp} + \frac{d_\text{st}}{2})\\
```

Comment:
Let's add a simple diagram to explain $\tau_u$. Something like this:

<img width="472" alt="image" src="https://github.com/Severson-Group/eMach/assets/7421348/fec6981d-16e0-49ec-9a71-0c4fa80f61da">

but not hand drawn... :-)

Can you add something?